lib/plugin.lazy.js

```diff
@@ -0,0 +1,139 @@
+import VueLib from 'vue'
+
+<% if (options.lazy.injectMock) { %>
+const apiMethods = <%= JSON.stringify(options.lazy.mockApiMethods)%>
+let delayedCalls = []
+let SentryMock = {}
+<% } %>
+let sentryReadyResolve
+let loaded = false
+
+<% if (options.lazy.injectMock) { %>
+const vueErrorHandler = VueLib.config.errorHandler
+
+VueLib.config.errorHandler = (err, vm, info) => {
```

Comment:
One other thing I've just realized is that having the global handler set makes hides the errors from the console (see https://github.com/vuejs/vue/blob/7186940143704acc4ec046132f6a56e9c983e510/src/core/util/error.js#L59).

See how Sentry re-triggers the console message in its handler:
https://github.com/getsentry/sentry-javascript/blob/820bd77cca08cc6f4e209b6cbf8dcf82ab3f3477/packages/integrations/src/vue.ts#L405-L410

Reply:
Addressed with a92de0a67cb5cecf125379c95bb2c77b48fdb2e2